src/System.ServiceModel.Security/tests/ServiceModel/SecureConversationSecurityTokenParametersTest.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
+using System.ServiceModel.Security.Tokens;
+using Infrastructure.Common;
+using Xunit;
+
+public static class SecureConversationSecurityTokenParametersTest
+{
+    [WcfTheory]
+    [InlineData(false)]
+    [InlineData(true)]
+    public static void RequireCancellation_Property_Is_Settable(bool value)
```

Comment:
Can you add a test which makes sure that the default value of RequireCancellation is the same as what is on .NET Framework.